Update documentation. Add information about git flexibility after task creation (#4401)
openvinotoolkit/cvat: site/content/en/docs/manual/basics/task-details.md

- The next actions are available on this page:
  1. Change the task’s title.
  1. Open `Actions` menu.
  1. Change issue tracker or open issue tracker if it is specified.
  1. Change labels (available only if the task is not related to the project).
     You can add new labels or add attributes for the existing labels in the Raw mode or the Constructor mode.
     By clicking `Copy` you will copy the labels to the clipboard.
  1. Assigned to — is used to assign a task to a person. Start typing an assignee’s name and/or
     choose the right person out of the dropdown list.
     In the list of users, you will only see the users of the [organization](/docs/manual/advanced/organization/)
     where the task is created.

  1. Dataset Repository
     - Repository link
     - Synchronization status with dataset repository.
       When you click on the status, the current annotation will be sent. It has several states:
       - Synchronized - task synchronized, that is, created a pull of requisites with an actual annotation file.
       - Merged - merged pull request with up-to-date annotation file.

>        - Synchronize - highlighted in red, there are sync issues.

Reviewer: ```suggestion
       - Synchronize - highlighted in red, annotations are not synced.
```
